tests/testthat/test-load.lcms.R

```diff
@@ -0,0 +1,23 @@
+patrick::with_parameters_test_that(
+  "test load.lcms reads mzML file",
+  {
+    # Arrange: Set up test inputs
+    testdata <- file.path("..", "testdata")
+    input_path <- file.path(testdata, filename)
+
+    # Act: Execute the function with the test inputs
+    sut <- load.lcms(input_path)
+
+    # Assert: Verify the function output matches expected results
+    # Check that the function returns an object of the expected type
+    testthat::expect_s3_class(sut, "tbl_df")
```

Comment:
Checking only for the expected type is not enough - I would also check some other things, like the number of rows etc. to see if all data is loaded correctly.

It is probably not necessary to store the whole read dataframe, so just checking for some key features like nrows, ncols, no NA values should be enough.

Reply:
I expanded the test as suggested ([e5aea30](https://github.com/RECETOX/recetox-aplcms/pull/216/commits/e5aea3062cbc4c688e8f71284af9bdd018c0d649)). Updated the links for the test files in the "input.txt" ([eff1434](https://github.com/RECETOX/recetox-aplcms/pull/216/commits/eff14342814c75ea3c98ee86acbec850cbdafd51)). Also I uploaded the mzxml file to gitlab umsa-files (PR is open).